ci/package_android_release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# This script's purpose is to automate the build + packaging steps for the pre-compiled Android verifier component.
+# It works with template files and directories inside the `android-release-support/` part of the repository to setup
+# a Maven local repository and then add the pre-compiled AAR file into it for distribution. The results of this packaging
+# are then included by `cargo` when publishing `rustls-platform-verifier-android`.
+
+set -euo pipefail
+
+if ! type mvn > /dev/null; then
+  echo "The maven CLI, mvn, is required to run this script."
+  echo "Download it from: https://maven.apache.org/download.cgi"
+  exit 1
+fi
+
+version=$(cat android-release-support/Cargo.toml | grep -m 1 "version = " | tr -d "version= " | tr -d '"')
+
+echo "Packaging v$version of the Android support component"
+
+pushd ./android
+
+./gradlew assembleRelease
+
+popd
+
+artifact_name="rustls-platform-verifier-release.aar"
+
+pushd ./android-release-support
+
+artifact_path="../android/rustls-platform-verifier/build/outputs/aar/$artifact_name"
+
+# Ensure no prior artifacts are present
+git clean -dfX "./maven/"
+
+cp ./pom-template.xml ./maven/pom.xml
+sed -i "" "s/\$VERSION/$version/" ./maven/pom.xml
```

Comment:
fwiw on my system w/ GNU sed 4.9 this bailed with the error:
```
sed: can't read s/$VERSION/0.1.0/: No such file or directory
```

It needed to be changed to remove the space between `-i` and `""`:

```bash
sed -i"" "s/\$VERSION/$version/" ./maven/pom.xml
```

Reply:
I noticed this as well when working on the automation PR. I'll cherry-pick the fix I made over there to this branch. There's an annoying incompatibility between macOS's `sed` and GNU's `sed`.

Reply:
> There's an annoying incompatibility between macOS's sed and GNU's sed.

argh, of course there is :dizzy_face:

Reply:
This should be fixed now.